lib/api/multipartDelete.js

```diff
@@ -122,14 +142,26 @@ function multipartDelete(authInfo, request, log, callback) {
                         bucket: bucketName,
                         byteLength: partSizeSum,
                     });
+                    return next(null, destBucket, location);
+                });
+        },
+        function ifMultipleBackend(destBucket, location, next) {
+            if (config.backends.data === 'multiple') {
+                return multipleBackendGateway.abortMPU(objectKey,
+                uploadId, location, destBucket, log, err => {
```

Comment:
No, the location should always be defined here, because the check is for the controlling location constraint, which defaults to either the bucket location or request endpoint location if the object is put without one :)